docs/topics/running/environment.md

```diff
@@ -62,32 +62,71 @@ verbose, valid log levels are `error`, `warn`/`warning`, `info`,
 
 The Ambassador Edge Stack make use of Redis for several purposes.  By
 default, all components of the Ambassador Edge Stack share a Redis
-connection pool; there will be a total of up to `REDIS_POOL_SIZE`
-connections to Redis.  If `REDIS_PERSECOND` is true, a second Redis
+connection pool.  If `REDIS_PERSECOND` is true, a second Redis
 connection pool is created (to a potentially different Redis instance)
 that is only used for per-second RateLimits; this second connection
 pool is configured by the `REDIS_PERSECOND_*` variables rather than
 the usual `REDIS_*` variables.
 
-If `REDIS_PASSWORD` (or `REDIS_PERSECOND_PASSWORD`) is non-empty, then
-it is used to `AUTH` to Redis immediately after the connection is
-established.  If `REDIS_USERNAME` (or `REDIS_PERSECOND_USERNAME`) is
-set, then that username is used with to log in as that user in the
-[Redis 6 ACL].  It is invalid to set a username without setting a
-password.  It is invalid to set a username with Redis 5 or lower.
-
-If `REDIS_TLS_ENABLED` (or `REDIS_PERSECOND_TLS_ENABLED`) is true,
-then TLS is used when communicating with Redis.  Setting
-`REDIS_TLS_INSECURE` (or `REDIS_PERSECOND_TLS_INSECURE`) to true
-disables TLS verification when doing so; alternatively, consider
-[installing Redis' self-signed certificate in to the Ambassador Edge
-Stack
-container](../../using/filters/#installing-self-signed-certificates).
-
-Note that when using a port name instead of a port number in a Go
-network address (as as in `REDIS_URL` or `REDIS_PERSECOND_URL`), the
-name refers a well-known port name in the container's `/etc/services`,
-and **not** to a Kubernetes port name.
+#### Redis layer 4 connectivity
```

Comment:
```suggestion
#### Redis layer 4 connectivity (L4)
```

It took a bit for me to realize that these 3 sections correspond to the tags that were listed in the table.  This makes it a little more explicit.